### ISSUE TITLE
send a command to a channel, similar to echo?

### ISSUE THREAD

**Issue by classabbyamp** [miaowware/qrm2#236]
```
?echocmd #bots rq ca b
    channel^    ^cmd+args
```

**Comment by classabbyamp** [comment]
related to #227, maybe add both as subcommands of echo?

**Comment by 0x5c** [comment]
Just enable parsing of commands on message by the `ClientUser`, there is an argument for that in `Bot()`

**Comment by classabbyamp** [comment]
yes there's that, but I'm also thinking about doing something like sending the cmd result without showing the command

**Comment by 0x5c** [comment]
Hum, then if you manage to get a `Message` object you could do `bot.process_commands(message)`.
